.github/workflows/release-please.yml

```
@@ -0,0 +1,16 @@
+name: Release Please
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)